button.js

```diff
@@ -11,18 +11,16 @@ function replaceWord() {
   fetch("https://wordsapiv1.p.rapidapi.com/words/?random=true", {
     method: "GET",
     headers: {
-      "x-rapidapi-key": "your-api-key-here",
-      "x-rapidapi-host": "api-host-here",
+      "x-rapidapi-host": "wordsapiv1.p.rapidapi.com",
+      "x-rapidapi-key": "keys",
     },
   })
     .then((response) => {
-      console.log(response);
-      response.json().then(() => {
-        /* 
-          Complete this handler function with code that populates an html 
-          element with the random word 
-        */
+      response.json().then((responseBody) => {
+        console.log(responseBody.word);
```

Comment:
console.log statements are useful for debugging but don't need to be left in the finished code

Reply:
I will take it out, Thanks

Reply:
When I take out the console.log(responseBody.word); the word won't display when I click the button. What can I do?

Reply:
I realize that I change the rapidapi-key for "keys".